cypress/component/ResultContainer.cy.tsx

```diff
@@ -0,0 +1,66 @@
+import ResultContainer from '../../src/components/ResultContainer';
+import { protectedResponse2 } from '../fixtures/mocked-responses';
+
+describe('ResultContainer', () => {
+  it('Displays a set of Result Cards, select all checkbox, disabled download result buttons, summary stats, and how to get data modal button', () => {
+    cy.mount(<ResultContainer result={protectedResponse2} />);
```

Comment:
pretty cool that this just works, i.e. that you only need to mount the parent here and react figures out how to get the children :tada:

Reply:
Indeed, no need to stub the children anymore.